packages/contentful--app-scripts/src/utils.ts

```diff
@@ -108,11 +108,17 @@ export function getEntityFromManifest<Type extends 'actions' | 'functions'>(type
       )}.`,
     );
 
+    const fieldMappingEvent = "graphql.field.mapping";
+    const queryEvent =  "graphql.query";
+
     const items = (manifest[type] as FunctionAppAction[] | ContentfulFunction[]).map((item) => {
       const allowNetworks = Array.isArray(item.allowNetworks)
         ? item.allowNetworks.map(stripProtocol)
         : [];
 
+      const accepts = 'accepts' in item ? Array.isArray(item.accepts) ? item.accepts : undefined : undefined;
```

Comment:
```suggestion
      const accepts = 'accepts' in item && Array.isArray(item.accepts) ? item.accepts : undefined;
```
Array.isArray(item.accepts) will return false if item.accepts is undefined